templates/test/tr_job_result_failedmodules.html.ep

```diff
@@ -21,8 +21,7 @@
     % }
 
     <span <%= Mojo::ByteStream->new($tooltip) %> class="failedmodule" >
-        <a href="<%= url_for('step', 'testid' => $jobid,
-                             'moduleid' => $failedmodule, 'stepid' => $step) %>">
+        <a href="<%= url_for('test', 'testid' => $jobid) . "?$failedmodule/steps/$step" %>">
```

Comment:
The step page is only used for the AJAX request now.
I would only redirect old users (with some to be created hack) but send the new users directly to the right page.
Especially as redirecting introduces a 2nd http request and might break browser history.